components/_patterns/00-base/_utilities.scss

```diff
@@ -1,14 +1,10 @@
-.more-link {
-  @include more-link;
-}
-
-// Drupal-specific
-.align-left {
-  float: left;
-  margin-right: .4em;
-}
-
-.align-right {
-  float: left;
-  margin-left: .4em;
-}
+// Possibly helpful for wysiwyg areas?
+// .align-left {
+//   float: left;
+//   margin-right: .4em;
+// }
+//
+// .align-right {
+//   float: left;
+//   margin-left: .4em;
+// }
```

Comment:
Do we want to keep unused code in the repo? I'd suggest we either make them available (uncomment them) or just get rid of the file.